notificationworkerlambda/topic-counter-cfn.yaml

```diff
@@ -0,0 +1,151 @@
+Parameters:
+  Stack:
+    Description: Stack name
+    Type: String
+    Default: mobile-notifications
+  App:
+    Description: Application name
+    Type: String
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+    - CODE
+    - PROD
+    Default: CODE
+  DeployBucket:
+    Description: Bucket where RiffRaff uploads artifacts on deploy
+    Type: String
+    Default: mobile-notifications-dist
+  VpcId:
+    Description: ID of the Notification VPC
+    Type: AWS::EC2::VPC::Id
+  VpcSubnets:
+    Description: Subnets to use in the VPC
+    Type: List<AWS::EC2::Subnet::Id>
+  VPCSecurityGroup:
+    Type: AWS::EC2::SecurityGroup::Id
+    Description: The default security group of the VPC
+  FullyQualifiedHandler:
+    Description: The full name of the handler, including path, class name and method
+    Type: String
+
+Resources:
+
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service:
+            - lambda.amazonaws.com
+          Action: sts:AssumeRole
+      Path: /
+      Policies:
+      - PolicyName: logs
+        PolicyDocument:
+          Statement:
+          - Effect: Allow
+            Action: logs:CreateLogGroup
+            Resource: !Sub "arn:aws:logs:eu-west-1:${AWS::AccountId}:*"
+          - Effect: Allow
+            Action:
+            - logs:CreateLogStream
+            - logs:PutLogEvents
+            Resource:
+            - !Sub "arn:aws:logs:eu-west-1:${AWS::AccountId}:log-group:/aws/lambda/${App}-${Stage}:*"
+      - PolicyName: VPC
+        PolicyDocument:
+          Statement:
+            Effect: Allow
+            Action:
+            - ec2:CreateNetworkInterface
+            - ec2:DescribeNetworkInterfaces
+            - ec2:DeleteNetworkInterface
+            Resource: "*"
+      - PolicyName: Conf
+        PolicyDocument:
+          Statement:
+          - Action: ssm:GetParametersByPath
+            Effect: Allow
+            Resource:
+              !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/notifications/${Stage}/workers
+      - PolicyName: Cloudwatch
+        PolicyDocument:
+          Statement:
+            Effect: Allow
+            Action: cloudwatch:PutMetricData
+            Resource: "*"
+      - PolicyName: S3
+        PolicyDocument:
+          Statement:
+          - Action:
+            - s3:PutObject
+            Effect: Allow
+            Resource: arn:aws:s3:::mobile-notifications-topics/*
+          - Action:  s3:ListBucket
+            Effect: Allow
+            Resource: arn:aws:s3:::mobile-notifications-topics
+
+  LambdaSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: !Sub Lambda to db in ${Stage}
+      SecurityGroupEgress:
+        - CidrIp: 0.0.0.0/0
+          FromPort: 443
+          IpProtocol: tcp
+          ToPort: 443
+      VpcId: !Ref VpcId
+
+  TopicCounterWorker:
```

Comment:
Is this duplicated in the worker?

Reply:
I forgot to take it out of the worker yaml. Where it was originally and I've now removed it from